upb/table.c

```diff
@@ -496,20 +500,20 @@ bool upb_strtable_insert(upb_strtable* t, const char* k, size_t len,
   tabkey = strcopy(key, a);
   if (tabkey == 0) return false;
 
-  hash = _upb_Hash(key.str.str, key.str.len);
+  hash = _upb_Hash_NoSeed(key.str.str, key.str.len);
```

Comment:
why did this go to NoSeed and not use wyhashseed?

Reply:
Wyhash has both "seed" and "salt". We still use Wyhash's salt, but the seed is generally only useful when hashing multiple things and wanting to propagate state from one call to the next.